virttest/utils_disk.py

```diff
@@ -1102,6 +1102,20 @@ def get_parts_list(session=None):
     return parts
 
 
+def get_added_parts(session, old_parts):
```

Comment:
How about setting a default value for session? I think this function can also be used in localhost, just like get_parts_list(). What do you think?

Reply:
My original thought is that we will not test scenario to add a new disk to a host. So I only apply this function to operate in vm.

Reply:
Ok, I see. But I still think it's better to use the same way as get_parts_list(). Anyway, it's up to you!

Reply:
I would not like to write codes which we do not expect to use. However, I will be happy to enhance it once we need it in future. Thanks.

Reply:
Ok, got it.